src/js/main.js

```diff
@@ -367,11 +424,10 @@ Site.Doodle = {
 
     _this.keyframe++;
   },
-}
-
-jQuery(document).ready(function () {
-  'use strict';
+};
 
-  Site.init();
+Site.init();
 
-});
+function onYouTubeIframeAPIReady() {
```

Comment:
is this function coming from the api library? you should specify that in a comment